packages/web-app-files/tests/unit/components/Search/Preview.spec.js

```diff
@@ -0,0 +1,51 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import DesignSystem from 'owncloud-design-system'
+
+import Preview from '@files/src/components/Search/Preview.vue'
+
+const localVue = createLocalVue()
+localVue.use(DesignSystem)
+
+const searchResult = {
+  id: 1234,
+  data: {
+    name: 'lorem.txt'
+  }
+}
+
+describe('Preview component', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  const spyTriggerDefaultAction = jest
+    .spyOn(Preview.mixins[0].methods, '$_fileActions_triggerDefaultAction')
+    .mockImplementation()
+
+  let wrapper
+  beforeEach(() => {
+    wrapper = getShallowWrapper()
+  })
+
+  it('should set correct props on oc-resource component', () => {
+    const ocResource = wrapper.find('oc-resource-stub')
+
+    expect(ocResource.exists()).toBeTruthy()
+    expect(ocResource.props().resource).toMatchObject(searchResult.data)
+  })
+  it('should call "$_fileActions_triggerDefaultAction" on click event', async () => {
+    const searchPreview = wrapper.find('.files-search-preview')
```

Comment:
There seems to be no use for this selector elsewhere, but it's still better to access them from `vars`